connect/logger/LoggerConfig.hx

```diff
@@ -34,19 +34,33 @@ class LoggerConfig extends Base {
      *  One of: `Logger.LEVEL_ERROR`, `Logger.LEVEL_INFO`, `Logger.LEVEL_DEBUG`.
      * @return `this` instance to support a fluent interface.
      */
-    public function level(level: Int): LoggerConfig {
+    public function level(level:Int):LoggerConfig {
         this.level_ = level;
         return this;
     }
 
+    /**
+     * Sets the logging level. Default is `Logger.LEVEL_INFO`.
+     * @param level Level of log.
+     *  One of: `ERROR`, `INFO`, `DEBUG`.
+     * @return `this` instance to support a fluent interface.
+     */
+    public function levelName(level:String):LoggerConfig {
+        this.level_ = Logger.LEVEL_INFO;
+        if (Std.is(level, String) && this.levelTranslation.exists(level)) {
```

Comment:
Although the user could send any type as argument in a dynamic language, we are not performing the `Std.is` checks on the SDK when an specific type is specified for a parameter. Since the SDK documentation will indicate that the type of the parameter is `String`, it is acceptable to not check and blame it on the user if he sends anything else.